### PR TITLE
reset modal now prompts user to uninstall and reinstall app

### PR DESCRIPTION
**Pull request by valpinkman**
Remove the reset app function and instead prompt the user to uninstall and reinstall app

### Type

Improvement

### Context

LL-1327

### Parts of the app affected / Test plan

Settings => Help => Reset Ledger Live

### CODE REVIEW

**Comment by gre** [comment]
@valpinkman some eslint failure

File: src/locales/en/common.json [hunk 1]
```diff
@@ -373,8 +373,7 @@
     }
   },
   "reset": {
-    "description": "Reset all Ledger Live data, including accounts and settings? The private keys providing access to your crypto assets remain secure on your device and Recovery sheet.",
-    "button": "Reset"
+    "description": "To remove Ledger Live app data including accounts and settings, please uninstall and reinstall it"
```
Comment (gre) [inline src/locales/en/common.json]: shouldn't we remove the description as well?

**Comment by gre** [comment]
@valpinkman can you screenshot what the new settings look like to show to @dasilvarosa for wording validation? thx

**Comment by valpinkman** [comment]
Will do right now

**Comment by valpinkman** [comment]
IMG_804BDD4DC38E-1

**Comment by Arnaud97234** [comment]
@gre @valpinkman Could we add a "Quit app" button in the modal ?

**Comment by dasilvarosa** [comment]
Did not get a notification before it was merged. 

Would suggest: Please uninstall the app from your phone to remove Ledger Live data, including accounts and settings.

**Pull request merged**